tests/testTriggers.go

```diff
@@ -4719,6 +4721,57 @@ func TriggerAppTasksDown(contexts *[]*scheduler.Context, recordChan *chan *Event
 	})
 }
 
+// TriggerValidateDeviceMapperCleanup validate device mapper device cleaned up for FA setup
+func TriggerValidateDeviceMapperCleanup(contexts *[]*scheduler.Context, recordChan *chan *EventRecord) {
+	defer ginkgo.GinkgoRecover()
+	event := &EventRecord{
+		Event: Event{
+			ID:   GenerateUUID(),
+			Type: ValidateDeviceMapper,
+		},
+		Start:   time.Now().Format(time.RFC1123),
+		Outcome: []error{},
+	}
+	defer func() {
+		event.End = time.Now().Format(time.RFC1123)
+		*recordChan <- event
+	}()
+	logrus.Infof("Validating the deviceMapper devices cleaned up or not")
+	Step("Match the devicemapper devices in each node if it matches the expected count or not ", func() {
+		pureVolAttachedMap, err := Inst().V.GetNodePureVolumeAttachedCountMap()
+		if err != nil {
+			logrus.Error(err)
+			UpdateOutcome(event, err)
+		}
+
+		for _, n := range node.GetWorkerNodes() {
+			logrus.Infof("Validating the node: %v", n.Name)
+			expectedDevMapperCount := 0
+			storageNode, err := Inst().V.GetPxNode(&n)
+			if err != nil {
+				logrus.Error(err)
+				UpdateOutcome(event, err)
+			}
+			expectedDevMapperCount += len(storageNode.Disks)
```

Comment:
if there is a error, storagenode will be nil and this might go into panic.
I think we need to skip this if there an error

Reply:
why we have to skip error here? Need to understand more from you..

Reply:
> why we have to skip error here? Need to understand more from you..

we do not skip error, we might need to skip the rest of the logic if there is an error in getting storage node as either storagenode will be nil or len(storagenode.disks) will be 0